torchvision/datasets/stl10.py

```diff
@@ -65,10 +65,12 @@ def __init__(
         self.labels: Optional[np.ndarray]
         if self.split == "train":
             self.data, self.labels = self.__loadfile(self.train_list[0][0], self.train_list[1][0])
+            self.labels = cast(np.ndarray, self.labels)
```

Comment:
We already have `self.labels: Optional[np.ndarray]` above, should we remove one of these?

Reply:
I missed that. Let me have a look.